src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/5.sql

```diff
@@ -0,0 +1,1869 @@
+-- Style guide: please see: https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md
+
+/*************************************************************
+    Schema Version
+**************************************************************/
+
+INSERT INTO dbo.SchemaVersion
+VALUES
+    (4, 'started')
```

Comment:
Should this be 5?

Reply:
5.diff.sql would need that too, maybe it's intentional? Just making sure

Reply:
Good catch! Thank yo. It seems to be added by the tool when applying the diff. (I don't think should be needed for the full sql file, @rbans96, what do you think)?

Reply:
Good point! Since dropping of database objects are removed from the full sql file from version 4 and onwards, we can insert the version in the same way as we do for diff files by the tool or SchemaUpgradeRunner.cs(for automatic upgrades). Probably we can plan to do this change for the next version scripts(>=6).

Reply:
Thanks. Opened https://github.com/microsoft/healthcare-shared-components/issues/113